src/components/User.tsx

```diff
@@ -40,10 +49,24 @@ const User = () => {
 			);
 		default:
 			return (
-				<div className="App-navbar__item">
-					<UserIconSolid />
-					{oidcUser.name}
-				</div>
+				<Dropdown name="navbar-dropdown">
+					<DropdownTrigger type="navbar-item">
+						<UserIconSolid />
+						{oidcUser.name}
+					</DropdownTrigger>
+					<DropdownMenu align="right">
+						<DropdownMenuButton
+							icon={<PowerIcon />}
+							className="logout"
+							onClick={() => {
+								logout('/').catch(logger.error);
```

Comment:
This is great, works exactly as expected on my local branch, whatever page I may be on when I click logout i get redirected to the login screen